src/reference-data/defaults/preassigned-fare-products.json

```diff
@@ -407,5 +407,119 @@
+    ],
+    "distributionChannel": ["app"],
+    "id": "ATB:PreassignedFareProduct:5f61ee14",
+    "limitations": {
+      "userProfileRefs": [
+        "ATB:UserProfile:1ae560ba",
+        "ATB:UserProfile:d3e4ec09",
+        "ATB:UserProfile:8ee842e3",
+        "ATB:UserProfile:2832569f"
+      ]
+    },
+    "name": {"lang": "nob", "value": "7-dagersbillett hurtigbåt"},
+    "type": "boat-period",
+    "version": "ATB:Version:b158a267-314f-4176-8462-aee1e16067f3"
+  },
+  {
+    "alternativeNames": [
+      {"lang": "nno", "value": "30-dagersbillett hurtigbåt"},
+      {"lang": "eng", "value": "Monthly pass express boat"}
+    ],
+    "distributionChannel": ["app"],
+    "id": "ATB:PreassignedFareProduct:7fb92850",
+    "limitations": {
+      "userProfileRefs": [
+        "ATB:UserProfile:1ae560ba",
+        "ATB:UserProfile:d3e4ec09",
+        "ATB:UserProfile:8ee842e3",
+        "ATB:UserProfile:2832569f"
+      ]
+    },
+    "name": {"lang": "nob", "value": "30-dagersbillett hurtigbåt"},
+    "type": "boat-period",
+    "version": "ATB:Version:4c60d382-fd7f-4c83-b606-94c90a799c9a"
+  },
+  {
+    "alternativeNames": [
+      {"lang": "nno", "value": "60-dagersbillett båt"},
+      {"lang": "eng", "value": "60 days pass boat"}
+    ],
+    "distributionChannel": ["app"],
+    "id": "ATB:PreassignedFareProduct:450812f3",
+    "limitations": {
+      "userProfileRefs": [
+        "ATB:UserProfile:1ae560ba",
+        "ATB:UserProfile:d3e4ec09",
+        "ATB:UserProfile:8ee842e3",
+        "ATB:UserProfile:2832569f"
+      ]
+    },
+    "name": {"lang": "nob", "value": "60-dagersbillett båt"},
+    "type": "boat-period",
+    "version": "ATB:Version:7bd72f16-e107-49f3-be43-61f024241e9c"
+  },
+  {
+    "alternativeNames": [
+      {"lang": "nno", "value": "90-dagersbillett båt"},
+      {"lang": "eng", "value": "90 days pass boat"}
+    ],
+    "distributionChannel": ["app"],
+    "id": "ATB:PreassignedFareProduct:5d5f9574",
+    "limitations": {
+      "userProfileRefs": [
+        "ATB:UserProfile:1ae560ba",
+        "ATB:UserProfile:d3e4ec09",
+        "ATB:UserProfile:8ee842e3",
+        "ATB:UserProfile:2832569f"
+      ]
+    },
+    "name": {"lang": "nob", "value": "90-dagersbillett båt"},
+    "type": "boat-period",
+    "version": "ATB:Version:36144e69-e2ee-40c4-b715-706221701938"
+  },
+  {
+    "alternativeNames": [
+      {"lang": "nno", "value": "180-dagers billett båt"},
+      {"lang": "eng", "value": "180 days pass boat"}
+    ],
+    "distributionChannel": ["app"],
+    "id": "ATB:PreassignedFareProduct:ccb344d3",
+    "limitations": {
+      "userProfileRefs": [
+        "ATB:UserProfile:1ae560ba",
+        "ATB:UserProfile:d3e4ec09",
+        "ATB:UserProfile:8ee842e3",
+        "ATB:UserProfile:2832569f"
+      ]
+    },
+    "name": {"lang": "nob", "value": "180-dagers billett båt"},
+    "type": "boat-period",
+    "version": "ATB:Version:500a2e78-db4d-42b2-aa42-d869e586d18a"
```

Comment:
@jorelosorio Could you please add minimumVersion for the boat tickets with the value 1.39? This to avoid that 1.38 gets a half done solution.

Reply:
Added